### ISSUE TITLE
is possible to filter by tag?

### ISSUE THREAD

**Issue by tkart38** [hissy/addon_ajax_page_list#2]
hi is possible to filter the page list by tags?
i see this code? 
($parameters[0] == 'tag') {
                $bv->controller->action_filter_by_tag($parameters[1]);

how can i set the tags for make the filter?
thank in advances

**Comment by hissy** [comment]
Yes possible, but this example package is very old, I don't know it works on current concrete5 version

**Comment by tkart38** [comment]
I've installed  it on 5.7 and all work fine.
But how Can i pass the parameter for filter by tag?
Thanks
